packages/test-config/jest-unit.config.common.ts

```diff
@@ -22,6 +22,7 @@ const config: Partial<Config.InitialProjectOptions> = {
             },
         ],
     },
+    transformIgnorePatterns: [],
```

Comment:
This is here to make Jest work, because `@noble/ed25519` is an ESM-only package. No idea why it works.